doc/usermanual/CMakeLists.txt

```diff
@@ -9,8 +9,10 @@ message(STATUS ${IMAGES_PATHS})
 
 if(BUILD_USERMANUAL)
 	add_custom_target(usermanual ALL)
+	add_custom_target(api ALL)
```

Comment:
agreed, renaming to luaapi, 

however, only translated UM are added to the UM target and I did the same... I can change that, but i'd rather stay consistant with UM